requirements/extra.txt

```diff
@@ -7,4 +7,4 @@ torchtext>=0.3.1, <0.7  # TODO: temporary fix fix for compatibility
 onnx>=1.7.0
 onnxruntime>=1.3.0
 hydra-core>=1.0
-https://github.com/facebookresearch/fairscale/archive/8e85ce8c93569017521d92ceb78dba2c57c955a0.zip # TODO temporary fix till release version
+https://github.com/tchaton/fairscale/archive/dev.zip
```

Comment:
@tchaton shouldn't this be a link to one of Lightning's repo?

Reply:
We didn't want to fork fairscale yet. We will update this when the blog post are out.

Reply:
@tchaton I agree, but Jirka brought up a good point. If at any moment the fairscale repo is lost under your name, it breaks the PyPi package associated with this release (when you install extras). It might be on par better to fork and add the branch there

Reply:
it shall not be part of the release, see https://github.com/PyTorchLightning/pytorch-lightning/pull/5026/files#r538864886

Reply:
I would rather fork it and push it to pypi (test one)

Reply:
Yeah, let's do that !

Reply:
let's do what?
a) lightning fork and maintain the repo
b) push to test pypi
c) anything else...